rules/RESPONSE-951-DATA-LEAKAGES-SQL.conf

```
@@ -360,7 +360,7 @@ SecRule RESPONSE_BODY "@rx (?i)P(?:ostgreSQL(?: query failed:|.{1,20}ERROR)|G::[
     setvar:'tx.outbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}',\
     setvar:'tx.sql_injection_score=+%{tx.critical_anomaly_score}'"
 
-SecRule RESPONSE_BODY "@rx (?i)(?:Warning.*sqlite_.*|Warning.*SQLite3::|SQLite/JDBCDriver|SQLite\.Exception|System\.Data\.SQLite\.SQLiteException)" \
+SecRule RESPONSE_BODY "@rx (?i)(?:Warning[^s]+sqlite_[^\r\n]*|Warning[^s]+SQLite3::|SQLite/JDBCDriver|SQLite\.Exception|System\.Data\.SQLite\.SQLiteException)" \
```

```suggestion
SecRule RESPONSE_BODY "@rx (?i)(?:Warning[^s]+sqlite_|Warning[^s]+SQLite3::|SQLite/JDBCDriver|SQLite\.Exception|System\.Data\.SQLite\.SQLiteException)" \
```

Same here, does it add for detection purposes, or is it for logging purposes only?